deploy/operator.yaml

```diff
@@ -37,59 +37,53 @@ spec:
           value: "true"
         - name: DEBUG
           value: "false"
+        - name: RELATED_IMAGE_PAM_KIESERVER_IMAGE_8.0.0
+          value: registry.connect.redhat.com/ibm-bamoe/bamoe-kieserver-rhel8:8.0.0
+        - name: RELATED_IMAGE_PAM_CONTROLLER_IMAGE_8.0.0
+          value: registry.connect.redhat.com/ibm-bamoe/bamoe-controller-rhel8:8.0.0
+        - name: RELATED_IMAGE_PAM_BC_IMAGE_8.0.0
+          value: registry.connect.redhat.com/ibm-bamoe/bamoe-businesscentral-rhel8:8.0.0
+        - name: RELATED_IMAGE_PAM_BC_MONITORING_IMAGE_8.0.0
+          value: registry.connect.redhat.com/ibm-bamoe/bamoe-businesscentral-monitoring-rhel8:8.0.0
+        - name: RELATED_IMAGE_PAM_SMARTROUTER_IMAGE_8.0.0
+          value: registry.connect.redhat.com/ibm-bamoe/bamoe-smartrouter-rhel8:8.0.0
+        - name: RELATED_IMAGE_PAM_PROCESS_MIGRATION_IMAGE_8.0.0
+          value: registry.connect.redhat.com/ibm-bamoe/bamoe-process-migration-rhel8:8.0.0
+        - name: RELATED_IMAGE_PAM_DASHBUILDER_IMAGE_8.0.0
+          value: registry.connect.redhat.com/ibm-bamoe/bamoe-dashbuilder-rhel8:8.0.0
+        - name: RELATED_IMAGE_OSE_CLI_IMAGE_8.0.0
+          value: registry.redhat.io/openshift4/ose-cli:v4.10
+        - name: RELATED_IMAGE_MYSQL_PROXY_IMAGE_8.0.0
+          value: registry.redhat.io/rhscl/mysql-80-rhel7:latest
+        - name: RELATED_IMAGE_POSTGRESQL_PROXY_IMAGE_8.0.0
+          value: registry.redhat.io/rhscl/postgresql-10-rhel7:latest
+        - name: RELATED_IMAGE_DATAGRID_IMAGE_8.0.0
+          value: registry.redhat.io/datagrid/datagrid-8-rhel8:1.1
+        - name: RELATED_IMAGE_BROKER_IMAGE_8.0.0
+          value: registry.redhat.io/amq7/amq-broker-rhel8:7.9
         - name: RELATED_IMAGE_PAM_KIESERVER_IMAGE_7.13.0
-          value: registry.redhat.io/rhpam-7/rhpam-kieserver-rhel8:7.13.0
+          value: registry.connect.redhat.com/ibm-bamoe/bamoe-kieserver-rhel8:7.13.0
         - name: RELATED_IMAGE_PAM_CONTROLLER_IMAGE_7.13.0
-          value: registry.redhat.io/rhpam-7/rhpam-controller-rhel8:7.13.0
+          value: registry.connect.redhat.com/ibm-bamoe/bamoe-controller-rhel8:7.13.0
```

Comment:
It is not clear to me why here we are using tag 7.13 and not 8.0.0 but it is probably right the one used.

Reply:
Probably all related images for 7.13 should be removed. These images does not exists.